### PR TITLE
Add closed_motions_count to group

### PR DESCRIPTION
**Pull request by HSalmon**
Addresses: https://github.com/loomio/loomio/issues/3334

### CODE REVIEW

File: spec/models/discussion_spec.rb [hunk 1]
```diff
@@ -47,6 +47,31 @@
...
+    end
+
+    it "returns a count of closed motions" do
+      expect(@discussion.reload.closed_motions_count).to eq 1
+    end
+
+    it "updates correctly after motion is closed" do
+      expect {
+        @discussion.motions.create(attributes_for(:motion).merge({ author: @user })).close!
+      }.to change { @discussion.reload.motions_count }.by(1)
```
Comment (gdpelican) [inline spec/models/discussion_spec.rb]: `@discussion.reload.closed_motions_count`?

File: spec/models/group_spec.rb [hunk 2]
```diff
@@ -94,7 +94,36 @@
           @discussion.destroy
         }.to change { @group.reload.motions_count }.by(-1)
       end
+    end
+
+    describe "#closed_motions_count" do
+      before do
+        @user = create(:user)
```
Comment (gdpelican) [inline spec/models/group_spec.rb]: Why not use the `let(:user) { create(:user) }` syntax we use most other places? ❔

File: spec/models/discussion_spec.rb [hunk 3]
```diff
@@ -47,6 +47,31 @@
 
   end
 
+  describe "#closed_motions_count" do
+    before do
+      @user = create(:user)
```
Comment (gdpelican) [inline spec/models/discussion_spec.rb]: (same here, why not `let(:user) { create(:user) }`?

**Comment by gdpelican** [comment]
Cool, a couple nits, but looks good! 🍏

**Comment by robguthrie** [comment]
nice work hannah

File: spec/models/group_spec.rb (hunk 2 above)
Comment (HSalmon) [inline spec/models/group_spec.rb]: I used this syntax because it's consistent with the other counter cache specs. Should I change it?

**Comment by gdpelican** [comment]
Great work @HSalmon !

**Pull request merged**